halo/attest/keeper/keeper.go

```diff
@@ -98,6 +98,16 @@ func (k *Keeper) RegisterProposalService(server grpc1.Server) {
 	types.RegisterMsgServiceServer(server, NewProposalServer(k))
 }
 
+// AttestTable returns the attestations ORM table.
+func (k *Keeper) AttestTable() AttestationTable {
+	return k.attTable
+}
+
+// SignatureTable returns the attestations ORM table.
+func (k *Keeper) SignatureTable() SignatureTable {
+	return k.sigTable
+}
+
```

Comment:
These getters should be for testing only. So move then to a `keeper_internal_test.go` file.  That means the external tests can access them, but they are available in production or to other packages.

Reply:
> but they are available in production or to other packages.
But anyone can copy the 5 lines from `keeper.New` and inspect the attestation table. Therefore we can't really achieve better security by removing the getters, right? 
Would you mind if the tests are in `keeper_test` package? I can still remove the getter and setter and create a separate helper func which creates these two ORM tables given a KV store, then I'll reuse this func between the keeper and the tests.